tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/documentation/DeprecatedAnnotationTest.java

```diff
@@ -310,6 +310,29 @@ public void testDeprecatedObjectMemberField() {
         }
     }
 
+    @Test(description = "Test @deprecated annotation for record field")
+    public void testDeprecatedRecordField() {
+        List<Record> records = testModule.records;
+        Record employeeRecord = null;
+
+        for (Record record : records) {
+            String reocrdName = record.name;
+            if ("Employee".equals(reocrdName)) {
+                employeeRecord = record;
+            }
+        }
+
+        Assert.assertNotNull(employeeRecord);
+        for (DefaultableVariable field : employeeRecord.fields) {
+            String fieldName = field.name;
+            if ("name".equals(fieldName)) {
+                testDeprecated(field);
+            } else if ("age".equals(fieldName)) {
+                testNonDeprecated(field);
+            }
+        }
```

Comment:
I would rather write a method that accepts `employeeRecord.fields` and the name of the field and returns a `DefaultableVariable` corresponding to the field. And then do the assertions.

e.g.,
```ballerina
List<DefaultableVariable> fields = employeeRecord.fields;
testDeprecated(getField(fields, "name"));
testNonDeprecated(getField(fields, "age"));
```

Reply:
Assume by mistake the source for this test is changed to have different field names. If that happens, given the way the test is written at the moment, neither will it fail nor will it test what we want to test.